### PR TITLE
Improve relayer logs

### PR DESCRIPTION
**Pull request by pm47**
We can be less repetitive given that we now have a `relayId` that we can
use to factor all info.

### CODE REVIEW

File: eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala [hunk 1]
```diff
@@ -74,11 +74,15 @@ object ChannelRelayer {
           Behaviors.receiveMessage {
             case Relay(channelRelayPacket) =>
               val relayId = UUID.randomUUID()
-              val channels: Map[ShortChannelId, Relayer.OutgoingChannel] = channelUpdates.get(channelRelayPacket.payload.outgoingChannelId) match {
-                case Some(channel) => node2channels.get(channel.nextNodeId).map(channelUpdates).map(c => c.channelUpdate.shortChannelId -> c).toMap
+              val nextNodeId_opt: Option[PublicKey] = channelUpdates.get(channelRelayPacket.payload.outgoingChannelId) match {
+                case Some(channel) => Some(channel.nextNodeId)
+                case None => None
+              }
```
Comment (t-bast) [inline eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala:80]: nit:
```suggestion
              val nextNodeId_opt: Option[PublicKey] = channelUpdates.get(channelRelayPacket.payload.outgoingChannelId).map(_.nextNodeId)
```

File: eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala [hunk 2]
```diff
@@ -94,11 +98,13 @@ object ChannelRelayer {
             case WrappedLocalChannelUpdate(LocalChannelUpdate(_, channelId, shortChannelId, remoteNodeId, _, channelUpdate, commitments)) =>
               context.log.debug(s"updating local channel info for channelId=$channelId shortChannelId=$shortChannelId remoteNodeId=$remoteNodeId channelUpdate={} commitments={}", channelUpdate, commitments)
               val channelUpdates1 = channelUpdates + (channelUpdate.shortChannelId -> Relayer.OutgoingChannel(remoteNodeId, channelUpdate, commitments))
-              apply(nodeParams, register, channelUpdates1, node2channels.addOne(remoteNodeId, channelUpdate.shortChannelId))
+              val node2channels1 = node2channels.addOne(remoteNodeId, channelUpdate.shortChannelId)
+              apply(nodeParams, register, channelUpdates1, node2channels1)
```
Comment (t-bast) [inline eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala:102]: Why did you make it an explicit variable?
Is it used differently in feature branches?

File: eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala (hunk 1 above)
Comment (pm47) [reply eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala:80]: The format I went with allows for easier merge in feature branches.

File: eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala (hunk 2 above)
Comment (pm47) [reply eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala:102]: It is more consistent than doing the operation in the function call.

File: eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala (hunk 1 above)
Comment (t-bast) [reply eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala:80]: Ok, got it!

**Review by t-bast — APPROVED** [review]

**Pull request merged**